generators/add-web-assets/exc-react/index.js

```diff
@@ -73,11 +78,15 @@ class ExcReactGenerator extends Generator {
       true
     )
     // add env variable to load ui in exc shell
+    const launchPrefix = getCliEnv() === STAGE_ENV
+      ? 'https://experience-stage.adobe.com/?devMode=true#/custom-apps/?localDevUrl='
+      : 'https://experience.adobe.com/?devMode=true#/custom-apps/?localDevUrl='
+
     utils.appendOrWrite(
       this,
       this.destinationPath(dotenvFilename),
       `## URL prefix used to run your application in the Adobe Experience Cloud Shell
-AIO_LAUNCH_URL_PREFIX="https://experience.adobe.com/?devMode=true#/custom-apps/?localDevUrl="
+AIO_LAUNCH_URL_PREFIX="${launchPrefix}"
```

Comment:
Should we remove this one from the .env, in https://github.com/adobe/aio-cli-plugin-app/pull/396 you already set the default.
If someone inits an app in stage and then wants to switch to prod he needs to edit that line in the `.env`. If we don't include it then app plugin will be able to set the correct default

Reply:
I agree. These kind of overrides could just be documented instead of actually putting in .env.

Reply:
👍 for doc so then feel free to remove the env var 🙂